graphql-server/src/utils/ogm.ts

```diff
@@ -1,123 +1,36 @@
 import { GraphQLOptionsArg, GraphQLWhereArg, OGM } from "@neo4j/graphql-ogm";
-import { sortArrayByNestedField } from "./flattening";
-import { SortDirection } from "../generated/graphql";
+import { ApolloServerContext } from "./servers";
+//import { SamplesQueryResult } from "./dataloader";
+import { runQuery } from "../schemas/neo4j";
 
 const MAX_ROWS = 500;
 
 export async function querySamplesList(
   ogm: OGM,
   where: GraphQLWhereArg,
```

Comment:
I think we have the oncotree data here, we can use it to set cancerType and cancerTypeDetailed.